### PR TITLE
-MsbException introduce

### PR DESCRIPTION
**Pull request by anstr**
-Refactoring

### CODE REVIEW

File: amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpConsumerAdapter.java [hunk 1]
```diff
@@ -23,7 +23,9 @@
      * @param topic - a topic name associated with the adapter
      * @param consumerThreadPool contains incoming messages wrapped as tasks for further processing. Parameters of this thread pool determine degree of
      *                           parallelism of incoming message processing
+     * @throws ChannelException if some problems during setup channel from ActiveMQ connection were occurred
```
Comment (vso-tc) [inline amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpConsumerAdapter.java]: ActiveMQ????

File: amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpProducerAdapter.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
     /**
      * The constructor.
      * @param topic - a topic name associated with the adapter
+     * @throws ChannelException if some problems during setup channel from ActiveMQ connection were occurred
```
Comment (vso-tc) [inline amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpProducerAdapter.java]: ActiveMQ????

File: core/src/main/java/io/github/tcdl/adapters/AdapterFactoryLoader.java [hunk 3]
```diff
@@ -24,13 +30,13 @@ public AdapterFactory getAdapterFactory() {
             Class clazz = Class.forName(adapterFactoryClassName);
             adapterFactory = clazz.newInstance();
         } catch (ClassNotFoundException e) {
-            throw new RuntimeException("The required MSB Adapter Factory '" + adapterFactoryClassName + "' is not supported.", e);
+            throw new AdapterFactoryNotSupportedException("The required MSB Adapter Factory '" + adapterFactoryClassName + "' is not supported.", e);
```
Comment (vso-tc) [inline core/src/main/java/io/github/tcdl/adapters/AdapterFactoryLoader.java]: I would argue these exceptions are too fine-grained

File: amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpConsumerAdapter.java (hunk 1 above)
Comment (anstr) [inline amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpConsumerAdapter.java]: Will be fixed.

File: amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpProducerAdapter.java (hunk 2 above)
Comment (anstr) [inline amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpProducerAdapter.java]: Will be fixed.

**Pull request merged**